applications/robot-motion-control/Makefile

```diff
@@ -14,6 +14,9 @@ ROBOT_ID ?= 1
 # Platform
```

Comment:
trailing 'a' in commit title ` applications/robot-motion-control: convert odometry config to inline parametersa`

Reply:
Moreover commit message indentation seems wrong:
<img width="648" height="309" alt="image" src="https://github.com/user-attachments/assets/e92df010-14d5-4c67-83cb-c91ceee21b51" />